lms/static/sass/views/_verification.scss

```diff
@@ -619,6 +619,44 @@
       }
     }
 
+    @mixin help {
+      margin-bottom: ($baseline*1.5);
+
+      &:last-child {
+        margin-bottom: 0;
+      }
+
+      .title {
+        @extend %hd-lv3;
+      }
+
+      .copy {
+        @extend %copy-detail;
+      }
+    }
+
+    .requirements {
+      margin-bottom: 10px;
+      @include outer-container;
+
+      .title {
+        @extend %hd-lv3;
+        margin-bottom: 10px;
+      }
+
+      .requirement {
+        @include help();
+
+        border: 1px solid #e5e5e5;
+        border-top: 5px solid #f4d78a;
+        padding: 10px;
+        box-shadow: 1px 1px 3px #888888;
+
+        // TODO Determine why this is not responsive.
```

Comment:
This is not a showstopper given that the page is not responsive, and probably not even usable on mobile clients due to lack of access to the camera.